src/TeamSpeak3QueryApi/Query/QueryClient.cs

```diff
@@ -31,128 +30,52 @@ public class QueryClient : IDisposable
 
         /// <summary>Gets the remote host of the Query API client.</summary>
         /// <returns>The remote host of the Query API client.</returns>
-        public string Host { get; }
+        public string Host { get; internal set; }
```

Comment:
Why internal?

Reply:
I don´t want that somebody outside this class could change the host. But the Ssh or Telnet client need to set the host and for that it must get more access.